client/pd_service_discovery.go

```diff
@@ -16,7 +16,6 @@ package pd
 
```

Comment:
Do we need to update 2019 to 2023?

Reply:
pd_service_discovery was renamed from base_client.go which has time mark 2019. Should I update it to 2023?

Reply:
no need